shared/src/proxies/practitioners/getPractitionersByNameProxy.ts

```diff
@@ -10,13 +10,14 @@ import { get } from '../requests';
  */
 export const getPractitionersByNameInteractor = (
```

Comment:
Could y'all add types to this method please?

Reply:
will do!